### PR TITLE
Fixed "tab-device-permission" Sample code errors

### PR DESCRIPTION
**Pull request by Pawan-MSFT**

### CODE REVIEW

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

File: samples/tab-device-permissions/nodejs/src/NavigatorPermission.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. 
+function NavigatorPermission() {
```
Comment (Wajeed-msft) [inline samples/tab-device-permissions/nodejs/src/NavigatorPermission.js:3]: Can we rename this function to CheckAndAlertForCameraPermission() ?

**Review by Wajeed-msft — APPROVED** [review]
Please rename the method and merge.

**Comment by ChetanSharma-msft** [comment]
**Executed build pipeline:** 
**Result:**

**Pull request merged**